frontend/src/components/OperationSteps/OperationSteps.tsx

```diff
@@ -146,22 +152,43 @@ const OperationSteps: FunctionComponent<OperationStepsProps> = (props) => {
     }
   };
 
+  const onDatasourceToggle = (data: string) => {
```

Comment:
```suggestion
  const onSelectDataSourceType = (dataSourceType: string) => {
```

Reply:
You also created a type for this, so why not import it instead of using string here?